geth/jail/cell_test.go

```diff
@@ -6,8 +6,10 @@ import (
 	"testing"
 	"time"
 
+	"fmt"
 	"github.com/robertkrimen/otto"
 	"github.com/stretchr/testify/suite"
+	"net"
```

Comment:
could you please move `fmt` and `net` into one section?

Reply:
Removed the new line, `gofmt` sorts them as;

```go
import (
	"fmt"
	"github.com/robertkrimen/otto"
	"github.com/stretchr/testify/suite"
	"net"
	"net/http"
	"net/http/httptest"
	"testing"
	"time"
)
```

Reply:
i think newline is needed between packages from stdlib and others

Reply:
gotcha! just updated it